### PR TITLE
 #kuzws-185 chat demo upgraded to kuzzle 1.0.0-beta.1

### PR DESCRIPTION
**Pull request by benoitvidis**

### CODE REVIEW

File: chat/README.md [hunk 1]
```diff
@@ -24,21 +24,23 @@ Simple tutorial project used to demonstrate Kuzzle basic features.
...
 ```
 
-Now, you can try to use the todolist at http://localhost
+Now, you can try to access the chat demo on http://localhost
 
 ### Stand alone
 
 * A running [kuzzle](https://github.com/kuzzleio/kuzzle) instance
-* (Optional) Edit the first line of the ``js/app.js`` file to configure your Kuzzle instance URL, if you don't opt for a default installation
+* (Optional) Edit the the ``js/config.js`` file to configure your Kuzzle instance URL, if you don't opt for a default installation
```
Comment (ballinette) [inline chat/README.md]: "the the" ;)

**Comment by ballinette** [comment]
+1

File: chat/102.html [hunk 2]
```diff
@@ -47,15 +47,13 @@
...
-
-
-  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.7/angular.min.js"></script>
-  <script type="text/javascript" src="bower_components/angular-scroll-glue/src/scrollglue.js"></script>
-  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.3/js/foundation.min.js"></script>
-  <script type="text/javascript" src="config.js"></script>
-  <script type="text/javascript" src="bower_components/socket.io-client/socket.io.js"></script>
-  <script type="text/javascript" src="bower_components/kuzzle-sdk/dist/kuzzle.min.js"></script>
+  <script src="bower_components/angular/angular.min.js"></script>
```
Comment (dbengsch) [inline chat/102.html]: All dependencies changes are not reflected in the bower file.

Comment (benoitvidis) [inline chat/102.html]: Ok, jquery added.

**Comment by dbengsch** [comment]
+1

**Comment by scottinet** [comment]
+1

**Pull request merged**